package.json

```
@@ -30,8 +30,8 @@
     "affected:format:fix": "nx affected -t format:fix"
   },
   "dependencies": {
-    "@epam/ai-dial-ui-kit": "^0.5.0-rc.11",
-    "@floating-ui/react": "^0.26.7",
+    "@epam/ai-dial-ui-kit": "^0.5.0-rc.33",
```

let's update to latest version and check that everything is ok

```suggestion
    "@epam/ai-dial-ui-kit": "^0.5.0-rc.45",
```